.github/workflows/release.yml

```diff
@@ -0,0 +1,60 @@
+name: Release CI
+on:
+  push:
+    branches:
+      - release
+jobs:
+  deploy:
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          profile: minimal
+
+      - name: Build Polkadot Archive
+        run: cargo build --release --manifest-path bin/polkadot-archive/Cargo.toml
+
+      - name: Prepare binary
+        run: |
+          mkdir -p release
+          strip bin/polkadot-archive/target/release/polkadot-archive
+
+      - name: Package for Debian
+        if: matrix.os == 'ubuntu-latest'
+        run: |
+          cargo install cargo-deb
+          cargo deb --manifest-path ./bin/polkadot-archive/Cargo.toml
+          mv ./bin/polkadot-archive/target/debian/*.deb ./release/polkadot-archive-linux.deb
+          tar -C ./bin/polkadot-archive/target/release/ -czvf ./release/polkadot-archive-linux.tar.gz ./polkadot-archive
+
+      - name: Package just binary
+        if: matrix.os == 'ubuntu-latest'
+        run: |
+          mv bin/polkadot-archive/target/release/polkadot-archive ./release/polkadot-archive
+
+      - name: Get changelog
+        id: changelog_reader
+        uses: mindsers/changelog-reader-action@v2
+        with:
+          validation_depth: 2
+          path: ./CHANGELOG.md
```

Comment:
How does it know which changelog entry? Do we need to put `version` here?

Reply:
> Look good from what I understand; just a question about how the changelog reader knows how to get the version. Is there a way to test without actually merging to the release branch?

Yeah, I tested this in my personal repo with the Substrate Archive Changelog: https://github.com/insipx/test-gh-action/releases/tag/v0.5.1

It will default to the latest version. The reason it knows is because this action knows the format of Keep A Changelog/it was made especially with that format in mind